### PR TITLE
[AKS] Create default role assignments for automatic sku

### PR DESCRIPTION
**Pull request by purpleran7**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command
Create three default role assignments for automatic sku clusters.

az aks create -g <rg>-n <cluster>--sku automatic

### General Guidelines

- [ X ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ X ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ X ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ X ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/aks-preview/azext_aks_preview/managed_cluster_decorator.py [hunk 1]
```diff
@@ -3696,6 +3702,35 @@ def postprocessing_after_mc_created(self, cluster: ManagedCluster) -> None:
                 is_called_from_extension=True,
             )
 
+        # Add role assignments for automatic sku
+        if cluster.sku.name == "Automatic":
```
Comment (chengliangli0918) [inline src/aks-preview/azext_aks_preview/managed_cluster_decorator.py:3706]: if cluster.sku is not None and cluster.sku.name=="Automatic"

**Review by chengliangli0918 — APPROVED** [review]

**Review by FumingZhang — COMMENTED** [review]
lgtm

Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=92958452&view=results) to validate the change.
- test_aks_automatic_sku

File: src/aks-preview/HISTORY.rst [hunk 2]
```diff
@@ -12,6 +12,13 @@ To release a new version, please select a new version number (usually plus 1 to
...
 
+3.0.0b12
+++++++++
+* Create three default role assignments for automatic sku clusters.
+* "Azure Kubernetes Service RBAC Cluster Admin"
+* "Azure Kubernetes Service RBAC Admin"
+* "Azure Kubernetes Service Cluster User Role"
+
 3.0.0b11
 +++++++
```
Comment (FumingZhang) [inline src/aks-preview/HISTORY.rst:23]:
```suggestion
    * "Azure Kubernetes Service RBAC Cluster Admin"
    * "Azure Kubernetes Service RBAC Admin"
    * "Azure Kubernetes Service Cluster User Role"

3.0.0b11
++++++++
```

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**